app/v1/controllers/ListAllBusinessesController.scala

```diff
@@ -72,12 +68,6 @@ class ListAllBusinessesController @Inject()(val authService: EnrolmentsAuthServi
 
           val response = Json.toJson(vendorResponse)
 
-          auditSubmission(ListAllBusinessesAuditDetail(
-            userDetails = request.userDetails,
-            nino = nino,
-            `X-CorrelationId` = serviceResponse.correlationId,
-            AuditResponse(OK, Right(Some(response)))))
-
           Ok(Json.toJson(vendorResponse))
```

Comment:
```suggestion
          Ok(response)
```
or delete `val response = ...`